plugins/UsersManager/Model.php

```diff
@@ -388,6 +389,15 @@ public function setTokenAuthWasUsed($tokenAuth, $dateLastUsed)
     {
         $token = $this->getTokenByTokenAuth($tokenAuth);
         if (!empty($token)) {
+
+            $lastUsage = strtotime($token['last_used']);
+            $newUsage = strtotime($dateLastUsed);
+
+            // update token usage only every 10 minutes
```

Comment:
might be good to add a comment why this is eg link to https://github.com/matomo-org/matomo/issues/16924 ?

Reply:
done.